modules/ebi-metagenomics/sanntis/main.nf

```diff
@@ -0,0 +1,50 @@
+process SANNTIS {
+    tag "$meta.id"
+    label 'process_high'
+
+    conda "${moduleDir}/environment.yml"
+    container "${ workflow.containerEngine == 'singularity' && !task.ext.singularity_pull_docker_container ?
+        'biocontainers/sanntis:0.9.3.5--pyhdfd78af_1' :
+        'biocontainers/sanntis:0.9.3.5--pyhdfd78af_1' }"
+
+    input:
+    tuple val(meta), path(interproscan_output), path(gbk)
+
+    output:
+    tuple val(meta), path("*_sanntis.gff"), emit: gff
+    path "versions.yml"           , emit: versions
+
+    when:
+    task.ext.when == null || task.ext.when
+
+    script:
+    def args = task.ext.args ?: ''
+    def prefix = task.ext.prefix ?: "${meta.id}"
+    """
+    grep -v "/protein_id=" ${gbk} > ${prefix}_prepped.gbk
+    sanntis \
+    --ip-file ${interproscan_output} \
```

Comment:
push these next 5 lines after sanntis one tab inside

Reply:
Done in https://github.com/EBI-Metagenomics/nf-modules/commit/ad6b4bbca5144c2f7659e3d4b8c1d3e1c648f5b4